src/tensor_ops.jl

```diff
@@ -59,7 +66,7 @@ Computes the norm of a tensor
 """
 Computes the outer product between two tensors `t1` and `t2`. Can also be called via the infix operator `⊗`.
 """
-@inline otimes{order, dim}(t1::AbstractTensor{order, dim}, t2::AbstractTensor{order, dim}) = otimes(promote(t1, t2)...)
+# @inline otimes{order, dim}(t1::AbstractTensor{order, dim}, t2::AbstractTensor{order, dim}) = otimes(promote(t1, t2)...)
```

Comment:
There were already more defined methods for the use cases I think so this one would never be used?